modules/uhttp/client/client.go

```diff
@@ -32,28 +32,35 @@ import (
 	"golang.org/x/net/context/ctxhttp"
 )
 
-var _serviceName string
+var (
+	_serviceName string
+	_defaultTrue = true
```

Comment:
nit: default seems to indicate that the default value is true. The default value is actually false. Not sure it needs a variable, you can just set inline.